js/src/Ice/Stream.d.ts

```diff
@@ -119,7 +119,22 @@ declare module "ice" {
         }
 
         class OutputStream {
-            constructor(communicator?: Ice.Communicator, encoding?: EncodingVersion, buffer?: Uint8Array);
+            constructor(encoding: EncodingVersion);
+            /**
+             * Constructs an empty output stream that uses the communicator's default encoding version and default class
```

Comment:
There is no "default class format" anymore. The format in this case is the CompactFormat.